src/main/kotlin/net/atos/zac/smartdocuments/rest/SmartDocumentsRESTService.kt

```diff
@@ -31,7 +32,7 @@ class SmartDocumentsRESTService @Inject constructor(
     @GET
     @Path("templates")
     fun listTemplates(): Set<RESTSmartDocumentsTemplateGroup> {
-        PolicyService.assertPolicy(loggedInUserInstance.get().isGeautoriseerdVoorAlleZaaktypen)
+        PolicyService.assertPolicy(loggedInUserInstance.get().roles.contains(ROLE_BEHEERDER))
```

Comment:
This seems weird. In other places we just use OPA policies to check for the correct role right? Surely here we also need to add a new OPA policy and then check on that?

Reply:
As part of this story I think we need to add a new OPA policy and update the policy documentation etc?

Reply:
Yes. That's my preference too. But it was tempting to see how this looks if we do not do that as it's a small change